sdk/tools/winesync/winesync.py

```diff
@@ -45,6 +46,9 @@ def __init__(self, module):
         self.wine_staging_repo = pygit2.Repository(self.wine_staging_src)
         self.reactos_repo = pygit2.Repository(self.reactos_src)
 
+        # the standard author signature we will use
+        self.winesync_author_signature = pygit2.Signature('winesync', 'ros-dev@reactos.org')
```

Comment:
I didn't need this in the past, why is it required now?

Reply:
This is to implement what I explain in commit https://github.com/reactos/reactos/pull/5898/commits/4748084ac1acc8b7e6a194d5880d17a7023d3474 :
>  [WINESYNC] Use the same 'winesync' author as for all the other commits the script generates.
>
> This now correctly sets the commit author as 'winesync' (before it was
the committer's developer name) for the following types of commits:
`[WINESYNC]: revert wine-staging patchset for <MODULE_NAME>`
and
`[WINESYNC]: <MODULE_NAME> is now in sync with wine-staging <WINE_TAG>`

while the other [WINESYNC] commits had already the commit author: 'winesync' .